docs/guide/groups.md

```diff
@@ -0,0 +1,47 @@
+Sometimes it can be necessary to create groups of repos, especially if the number
+of repos grows and if you have people in different teams work on different repositories.
+
+## Defining groups in the manifest
+
+The first step is to edit the `manifest.yml` file to describe the groups. Here's an
+example.
+
+```yaml
+repos:
+  - { src: one, url: git@gitlab.local:acme/one }
+  - { src: two, url: git@gitlab.local:acme/two }
+  - { src: three, url: git@gitlab.local:acme/three }
+
+groups:
+  default:
+    g1: [one, two]
+    g2: [three]
+```
+
+Here we define a `g1` group that contains the `one` and `two` repos, and a
+`g2` group that contains the `three` repo.
+
+## Using groups in `tsrc init`
+
+If you only need the repos in the `g1` group you can run:
+
+```
+tsrc init git@gitlab.local:acme/manifest --group g1
+```
+
+## Updating workspace configuration
+
+Alternatively, you can edit the `.tsrc/config.yml` file, like this:
+
+```yaml
+manifest_url: git@gitlab.local:acme/manifest.git
+manifest_branch: master
+repo_groups:
+- one   # <- specify the list of groups to use
```

Comment:
In previous examples, `one` is  a repo, not a group. Maybe use `g1` here?

Reply:
good catch